x-pack/test/detection_engine_api_integration/security_and_spaces/bundled_prebuilt_rules_package/install_latest_bundled_prebuilt_rules.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import fs from 'fs/promises';
+import path from 'path';
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  deleteAllPrebuiltRuleAssets,
+  deleteAllRules,
+  getPrebuiltRulesAndTimelinesStatus,
+} from '../../utils';
+import { BUNDLED_PACKAGE_DIR } from './config';
+import { downloadPackageFromFleet } from '../../utils/prebuilt_rules/download_prebuilt_rules_from_epr';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  const removeBundledPackages = async (bundledFleetPackageDir: string) => {
+    const files = await fs.readdir(bundledFleetPackageDir);
+
+    // Do not remove mock packages with version 99.0.0 and 99.0.1
+    // which are used to test the installation of prerelease packages
+    const downloadedFiles = files.filter((file) => !file.includes('99')) || [];
```

Comment:
This is hacky because we are preventing this function to remove the zip files for versions 99.0.0 and 99.0.1, which are not used in this file, but in `prerelease_packages.ts`. 

However, since both files share the same config file, they share the directory were the bundled Fleet packages are fetched from.

If we want this to be cleaner, one of these two test files shouls be moved to a new different directory, with its own config file, where we can set up a different `xpack.fleet.developer.bundledPackageLocation`.

I'm open to it but unsure about creating a multitude of directories and config files (many containing a single test file), so open to opinions.